src/pages/zzang/Nav/TopNav.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./TopNav.scss";
+
+
+function TopNav(){
+    return(
+    <div className="nav_wrap">
+        <div className="header">
+            <div className="nav_wrap">
+                <h1>WeBucks</h1>
+                <nav className="global_nav">
+                    <ul>COFFEE</ul>
+                    <ul>MENU</ul>
+                    <ul>STORE</ul>
+                    <ul>RESPONSIBILITY</ul>
+                    <ul>STARBUCKS REWARDS</ul>
+                    <ul>WHAT'S NEW</ul>
```

Comment:
```suggestion
               <ul>
                    <li>COFFEE</li>
                    <li>MENU</li>
                    <li>STORE</li>
                    <li>RESPONSIBILITY</li>
                    <li>STARBUCKS REWARDS</li>
                    <li>WHAT'S NEW</li>
               </ul>
```

이렇게 작성해도 되지 않을까하는 생각으로 코멘트 남깁니다.